src/pcapi/flask_app.py

```diff
@@ -110,7 +110,15 @@ def before_request() -> None:
 
 @app.after_request
 def log_request_details(response: flask.wrappers.Response) -> flask.wrappers.Response:
-    duration = round((time.perf_counter() - g.request_start) * 1000)  # milliseconds
+    try:
+        duration = round((time.perf_counter() - g.request_start) * 1000)  # milliseconds
```

Comment:
en mettant la définition des extras avant le try et en y supprimant le duration:
```suggestion
        extra["duration"] = round((time.perf_counter() - g.request_start) * 1000)  # milliseconds
```

Reply:
Ca me fait étrange d'avoir une duration de 0 si elle n'est pas définie